src/modules/users/model.ts

```diff
@@ -0,0 +1,35 @@
+import { DataTypes, Model } from 'sequelize';
+import { sequelize } from '../../loaders/sequelize';
+
+export interface UserModel extends Model {
+    id: string;
+    login: string;
+    password: string;
+    age: number;
+    isDeleted: boolean;
+}
+
+export const User = sequelize.define<UserModel>('Users', {
+    id: {
+        type: DataTypes.UUIDV4,
+        defaultValue: DataTypes.UUIDV4,
+        allowNull: false,
+        primaryKey: true
+    },
+    login: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    password: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    age: {
+        type: DataTypes.INTEGER,
+        allowNull: false
+    },
+    isDeleted: {
+        type: DataTypes.BOOLEAN,
```

Comment:
here we could add default value, like `default: false`, that means we do not need to pass this parameter when creating record

Reply:
+